public/service-worker.js

```diff
@@ -1,4 +1,103 @@
 /*
- This service worker doesn't actually do anything!
- I'm here just to support Google Chrome App Banner on Android
+ I'm here to support Google Chrome App Banner on Android
  */
+
+/*
+ Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
I'm uneasy about the license here. Why do we need it?

Reply:
I removed it.